stubs/commonmark/METADATA.toml

```diff
@@ -1,2 +1,7 @@
 version = "0.9.*"
 upstream_repository = "https://github.com/rtfd/commonmark.py"
+partial_stub = true
```

Comment:
Is this intentional?

Reply:
No. Post-merge conflict and I didn't notice
```suggestion
```